### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by phillipadsmith**
Adding options to revenue sources.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -190,15 +190,25 @@ <h5 id="msc1"></h5>
...
+                        Brand partnerships
+                        </option>
                         <option value="Classifieds">Classifieds</option>
                         <option value="Corporate sponsorship">
                           Corporate sponsorship
                         </option>
+                        <option value="Co-Productions">
+                            Co-Productions
+                        </option>
+                        <option value="courses">Courses/Online Education</option>
```
Comment (jasonmurtha) [inline index.html]: Can we update the option value here to 'Courses' or 'Courses/Online Education', as the value is also used as the label.

**Comment by phillipadsmith** [comment]
Doh! Good catch. Will do! :)

File: index.html (hunk 1 above)
Comment (phillipadsmith) [reply index.html]: Doh! Good catch. Will do! :)

Comment (phillipadsmith) [reply index.html]: Done!

**Review by jasonmurtha — APPROVED** [review]

**Pull request merged**